src/conic_form.jl

```diff
@@ -20,13 +31,21 @@ function +(c::ConicObj, d::ConicObj)
     if !haskey(new_obj, var)
       new_obj[var] = d[var]
     else
-      # .+ does not preserve sparsity
+      # .+ does not preserve sparsity 
       # need to override behavior
-      if size(new_obj[var]) == size(d[var])
-        new_obj[var] = new_obj[var] + d[var]
+      #for i in 1:2
```

Comment:
Looping is not a concern, it's just that we iterate 2 times and we would need to use an array variable which would be inconsistent with (x1,x2) notation that I have been using throughout.

The reason we have to create a new tuple (x1,x2) is because tuples are immutable and set index method is not defined on them.